src/Core/Components/CounterBadge/FluentCounterBadge.razor.cs

```diff
@@ -28,19 +46,19 @@ public partial class FluentCounterBadge : FluentBadge, IFluentComponentBase
 
     /// <summary>
     /// Gets or sets if the badge displays the count based on the specified lambda expression.
-    /// By default the badge only shows a count when it's not equal to 0.
+    /// By default the badge only shows a count when it's greater than 0.
```

Comment:
Why did you change that? If the Counter accept a negative value, some situations could be use it (I can't think of any yet, but the community is inventive 😁)

Reply:
Because the overflowcount attribute on thee web component only accepts a positive value. If it is set to 99, setting count to -10000 stills shows that then in the badge

Reply:
We just 'dictate' it is a positive value only. If they need one that shows negative values, they can create their own (subclass from Badge)